### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ericdill**
~~Supersedes #316~~

### CODE REVIEW

**Comment by ericdill** [comment]
This is good to go. I'll push the v0.0.4 tag whenever this gets merged

File: doc/resource/api/core.rst [hunk 1]
```diff
@@ -1,79 +0,0 @@
-====================
- :mod:`core` Module
-====================
```
Comment (danielballan) [inline doc/resource/api/core.rst]: I don't think this should have been nuked. I can't find a replacement elsewhere.

**Comment by danielballan** [comment]
Other that file that I don't think should have been removed, feel free to self-merge. Tagging suggestion:

```
git commit --allow-empty -m "REL: v0.0.4rc1"
git tag v0.0.4rc1
git push upstream --tags
```

If everything seems to work (check `__version__`) then repeat, dropping the rc1.

**Comment by ericdill** [comment]
good catch

**Pull request merged**